### PR TITLE
Ensure versions

### PR DESCRIPTION
**Pull request by zahodi**
- role vars are automatically included when vars/main.yml are present, therefore no need to have an include task
- defined repos for v1 and v5 versions of filebeat per os_version
- add conditionals to define repos based on the filebeat_version var

### CODE REVIEW

File: tasks/main.yml [hunk 1]
```diff
@@ -1,9 +1,4 @@
 ---
-- include_vars: "{{ item }}"
-  with_first_found:
-    - "../vars/{{ ansible_os_family }}.yml"
```
Comment (DavidWittman) [inline tasks/main.yml:4]: There's a reason why I did it this way; there are separate variables for Debian and Redhat.

edit: I see now that your changes flattened them into a single file.

Comment (zahodi) [reply tasks/main.yml:4]: I moved the separate variables into vars/main.yml and the logic now resides within the tasks/debian.yml and tasks/redhat.yml. This allows to define versions and move to filebeat 5.

**Comment by DavidWittman** [comment]
Thanks @zahodi. I modified the PR to use a ternary instead of the `set_fact` module, mostly just out of personal preference. I'm going to do some further testing around the version pinning for Redhat based distros (as you're doing already with Debian) and then get this merged in.

**Comment by DavidWittman** [comment]
Thanks again @zahodi!

**Pull request merged**